lib/grpc/rpc_error.ex

```diff
@@ -96,4 +151,72 @@ defmodule GRPC.RPCError do
   def exception(status, message) when is_integer(status) do
     %GRPC.RPCError{status: status, message: message}
   end
+
+  @spec exception(status :: Status.t() | atom(), message :: String.t(), details :: [Google.Protobuf.Any.t()]) :: t()
```

Comment:
Similar to above - I wouldn't use positional args since both `message` and `details` are optional.